langchain-core/src/tools/index.ts

```diff
@@ -212,7 +212,8 @@ export abstract class StructuredTool<
     let result;
     try {
       result = await this._call(parsed, runManager, config);
-    } catch (e) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Changes here seem fine but unrelated?

Reply:
forgot to remove when I deleted the error handling code from this PR.